packages/salesforce/src/Utils.js

```diff
@@ -134,3 +134,36 @@ export async function pollJobResult(conn, job, pollInterval, pollTimeout) {
     attempt++;
   }
 }
+
+export function formatResults(input) {
```

Comment:
The integration tests are cool but I would value unit tests against this function way more!

I won't block the PR for tests on this though (especially as I've requested a change). If you have time for a few tests that's great. If you don't, don't worry.

What I'm really trying to say is: consider unit testing before integration testing. Also when you write a nice clean standalone util function like this, it's a great candidate for unit tests!